### DISCUSSION TITLE
What is the minimum supported iOS version?

### DISCUSSION THREAD

**Discussion by jforaker** [software-mansion/react-native-screens#1178] [Q&A] [answered]
Hey thanks for the great lib... questions is What is the minimum supported iOS version? Is the lib tested on iOS with an iOS Target Deployment below 13? 
We are getting mysterious crashes in our production app with users below iOS 13, and just thought I'd ask here because its not clearly specified in the docs. Yes I see the Podspec but I don't believe thats entirely accurate.

Of note, I see this line in a big open source app that heavily relies on react native screens: https://github.com/rainbow-me/rainbow/blob/develop/src/helpers/isNativeStackAvailable.js so I'm wondering if they're on to something there...

Any insight appreciated, thanks.

**Comment by kacperkapusciak** [comment, marked as answer]
Hi @jforaker,
we aim to support versions >=iOS 11. 

iOS 13 introduced **many** internal changes within the system and it sometimes results in a different behavior.

Before the release, we test a bunch of predefined routes on a few system versions and the features we've recently added. Testing *every* feature on *every* configuration isn't physically possible with the amount of resources we operate in the open-source team. Also, we hardly ever test more than one minor version from a major iOS release. For example, if we take `12.4` and a bug only appears on `12.1` we won't catch that.

If you think something is a bug please [file an issue here](https://github.com/software-mansion/react-native-screens/issues/new/choose).

Thanks for understanding and take care
